slam/alarm_table_view.py

```diff
@@ -144,6 +149,73 @@ def __init__(
 
         self.alarmModel.layoutChanged.connect(self.update_counter_label)
 
+    def handleThresholdDisplay(self):
```

Comment:
this function is the same in table and tree view minus the way it grabs the current alarm-item.

not sure best place for this to live as a shared function